<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Test Report</title>
    <link href="assets/style.css" rel="stylesheet" type="text/css"/></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) { // eslint-disable-line no-redeclare
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function findAll(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sortColumn(elem) {
    toggleSortStates(elem);
    const colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    let key;
    if (elem.classList.contains('result')) {
        key = keyResult;
    } else if (elem.classList.contains('links')) {
        key = keyLink;
    } else {
        key = keyAlpha;
    }
    sortTable(elem, key(colIndex));
}

function showAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(showExtras);
}

function hideAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(hideExtras);
}

function showExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.remove('collapsed');
    expandcollapse.classList.remove('expander');
    expandcollapse.classList.add('collapser');
}

function hideExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.add('collapsed');
    expandcollapse.classList.remove('collapser');
    expandcollapse.classList.add('expander');
}

function showFilters() {
    const filterItems = document.getElementsByClassName('filter');
    for (let i = 0; i < filterItems.length; i++)
        filterItems[i].hidden = false;
}

function addCollapse() {
    // Add links for show/hide all
    const resulttable = find('table#results-table');
    const showhideall = document.createElement('p');
    showhideall.innerHTML = '<a href="javascript:showAllExtras()">Show all details</a> / ' +
                            '<a href="javascript:hideAllExtras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    findAll('.col-result').forEach(function(elem) {
        const collapsed = getQueryParameter('collapsed') || 'Passed';
        const extras = elem.parentNode.nextElementSibling;
        const expandcollapse = document.createElement('span');
        if (extras.classList.contains('collapsed')) {
            expandcollapse.classList.add('expander');
        } else if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add('collapsed');
            expandcollapse.classList.add('expander');
        } else {
            expandcollapse.classList.add('collapser');
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener('click', function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains('collapsed')) {
                showExtras(event.currentTarget);
            } else {
                hideExtras(event.currentTarget);
            }
        });
    });
}

function getQueryParameter(name) {
    const match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () { // eslint-disable-line no-unused-vars
    resetSortHeaders();

    addCollapse();

    showFilters();

    sortColumn(find('.initial-sort'));

    findAll('.sortable').forEach(function(elem) {
        elem.addEventListener('click',
            function() {
                sortColumn(elem);
            }, false);
    });
}

function sortTable(clicked, keyFunc) {
    const rows = findAll('.results-table-row');
    const reversed = !clicked.classList.contains('asc');
    const sortedRows = sort(rows, keyFunc, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    const thead = document.getElementById('results-table-head');
    document.getElementById('results-table').remove();
    const parent = document.createElement('table');
    parent.id = 'results-table';
    parent.appendChild(thead);
    sortedRows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName('BODY')[0].appendChild(parent);
}

function sort(items, keyFunc, reversed) {
    const sortArray = items.map(function(item, i) {
        return [keyFunc(item), i];
    });

    sortArray.sort(function(a, b) {
        const keyA = a[0];
        const keyB = b[0];

        if (keyA == keyB) return 0;

        if (reversed) {
            return keyA < keyB ? 1 : -1;
        } else {
            return keyA > keyB ? 1 : -1;
        }
    });

    return sortArray.map(function(item) {
        const index = item[1];
        return items[index];
    });
}

function keyAlpha(colIndex) {
    return function(elem) {
        return elem.childNodes[1].childNodes[colIndex].firstChild.data.toLowerCase();
    };
}

function keyLink(colIndex) {
    return function(elem) {
        const dataCell = elem.childNodes[1].childNodes[colIndex].firstChild;
        return dataCell == null ? '' : dataCell.innerText.toLowerCase();
    };
}

function keyResult(colIndex) {
    return function(elem) {
        const strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
            'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[colIndex].firstChild.data);
    };
}

function resetSortHeaders() {
    findAll('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    findAll('.sortable').forEach(function(elem) {
        const icon = document.createElement('div');
        icon.className = 'sort-icon';
        icon.textContent = 'vvv';
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove('desc', 'active');
        elem.classList.add('asc', 'inactive');
    });
}

function toggleSortStates(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        resetSortHeaders();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function isAllRowsHidden(value) {
    return value.hidden == false;
}

function filterTable(elem) { // eslint-disable-line no-unused-vars
    const outcomeAtt = 'data-test-result';
    const outcome = elem.getAttribute(outcomeAtt);
    const classOutcome = outcome + ' results-table-row';
    const outcomeRows = document.getElementsByClassName(classOutcome);

    for(let i = 0; i < outcomeRows.length; i++){
        outcomeRows[i].hidden = !elem.checked;
    }

    const rows = findAll('.results-table-row').filter(isAllRowsHidden);
    const allRowsHidden = rows.length == 0 ? true : false;
    const notFoundMessage = document.getElementById('not-found-message');
    notFoundMessage.hidden = !allRowsHidden;
}
</script>
    <h1>Yatra Report</h1>
    <p>Report generated on 25-Feb-2022 at 03:24:02 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v3.1.1</p>
    <h2>Environment</h2>
    <table id="environment">
      <tr>
        <td>Packages</td>
        <td>{"pluggy": "1.0.0", "py": "1.11.0", "pytest": "6.2.5"}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Windows-10-10.0.19043-SP0</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{"forked": "1.4.0", "html": "3.1.1", "metadata": "1.11.0", "xdist": "2.5.0"}</td></tr>
      <tr>
        <td>Python</td>
        <td>3.10.0</td></tr></table>
    <h2>Summary</h2>
    <p>3 tests ran in 115.70 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="passed">0 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="failed">3 failed</span>, <input checked="true" class="filter" data-test-result="error" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="error">0 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable" col="duration">Duration</th>
          <th class="sortable links" col="links">Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">testcases/test_search_flights.py::TestSearchAndVerifyFilter::test_search_flights_1_stop_1___Mumbai____JFKdfgxzfb____25_02_2022____1_Stop__</td>
          <td class="col-duration">20.85</td>
          <td class="col-links"><a class="url" href="https://www.yatra.com/" target="_blank">URL</a> </td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div><div><img src="testcases/test_search_flights.py_TestSearchAndVerifyFilter_test_search_flights_1_stop_1___Mumbai____JFKdfgxzfb____25_02_2022____1_Stop__.png" alt="screenshot" style="width:304px;height:228px;" onclick="window.open(this.src)" align="right"/></div></div>
            <div class="log">self = &lt;testcases.test_search_flights.TestSearchAndVerifyFilter testMethod=test_search_flights_1_stop_1___Mumbai____JFKdfgxzfb____25_02_2022____1_Stop__&gt;<br/>going_from_origin = &#x27;Mumbai&#x27;, going_to_dest = &#x27;JFKdfgxzfb&#x27;, going_depart_date = &#x27;25/02/2022&#x27;, stop_filter = &#x27;1 Stop&#x27;<br/><br/>    @data(*Utils.read_data_from_csv(&quot;C:\\Users\\Hp\\PycharmProjects\\TestFrameworkMentor\\testdata\\test_csv_data.csv&quot;))<br/>    @unpack<br/>    def test_search_flights_1_stop(self, going_from_origin, going_to_dest, going_depart_date, stop_filter):<br/>&gt;       search_flight = self.launch_page.searchFlight(going_from_origin, going_to_dest, going_depart_date)<br/><br/>testcases\test_search_flights.py:35: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>pages\yatra_launch_page.py:86: in searchFlight<br/>    self.enterDepartureDate(departuredate)<br/>pages\yatra_launch_page.py:69: in enterDepartureDate<br/>    self.getDepartureDate().click()<br/>..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webelement.py:80: in click<br/>    self._execute(Command.CLICK_ELEMENT)<br/>..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webelement.py:693: in _execute<br/>    return self._parent.execute(command, params)<br/>..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py:418: in execute<br/>    self.error_handler.check_response(response)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>self = &lt;selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x000001A196F54970&gt;<br/>response = {&#x27;status&#x27;: 400, &#x27;value&#x27;: &#x27;{&quot;value&quot;:{&quot;error&quot;:&quot;element click intercepted&quot;,&quot;message&quot;:&quot;element click intercepted: Element ...\n\\tRtlGetAppContainerNamedObjectPath [0x77C37A9E+286]\\n\\tRtlGetAppContainerNamedObjectPath [0x77C37A6E+238]\\n&quot;}}&#x27;}<br/><br/>    def check_response(self, response: Dict[str, Any]) -&gt; None:<br/>        &quot;&quot;&quot;<br/>        Checks that a JSON response from the WebDriver does not have an error.<br/>    <br/>        :Args:<br/>         - response - The JSON response from the WebDriver server as a dictionary<br/>           object.<br/>    <br/>        :Raises: If the response contains an error message.<br/>        &quot;&quot;&quot;<br/>        status = response.get(&#x27;status&#x27;, None)<br/>        if not status or status == ErrorCode.SUCCESS:<br/>            return<br/>        value = None<br/>        message = response.get(&quot;message&quot;, &quot;&quot;)<br/>        screen: str = response.get(&quot;screen&quot;, &quot;&quot;)<br/>        stacktrace = None<br/>        if isinstance(status, int):<br/>            value_json = response.get(&#x27;value&#x27;, None)<br/>            if value_json and isinstance(value_json, str):<br/>                import json<br/>                try:<br/>                    value = json.loads(value_json)<br/>                    if len(value.keys()) == 1:<br/>                        value = value[&#x27;value&#x27;]<br/>                    status = value.get(&#x27;error&#x27;, None)<br/>                    if not status:<br/>                        status = value.get(&quot;status&quot;, ErrorCode.UNKNOWN_ERROR)<br/>                        message = value.get(&quot;value&quot;) or value.get(&quot;message&quot;)<br/>                        if not isinstance(message, str):<br/>                            value = message<br/>                            message = message.get(&#x27;message&#x27;)<br/>                    else:<br/>                        message = value.get(&#x27;message&#x27;, None)<br/>                except ValueError:<br/>                    pass<br/>    <br/>        exception_class: Type[WebDriverException]<br/>        if status in ErrorCode.NO_SUCH_ELEMENT:<br/>            exception_class = NoSuchElementException<br/>        elif status in ErrorCode.NO_SUCH_FRAME:<br/>            exception_class = NoSuchFrameException<br/>        elif status in ErrorCode.NO_SUCH_WINDOW:<br/>            exception_class = NoSuchWindowException<br/>        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:<br/>            exception_class = StaleElementReferenceException<br/>        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:<br/>            exception_class = ElementNotVisibleException<br/>        elif status in ErrorCode.INVALID_ELEMENT_STATE:<br/>            exception_class = InvalidElementStateException<br/>        elif status in ErrorCode.INVALID_SELECTOR \<br/>                or status in ErrorCode.INVALID_XPATH_SELECTOR \<br/>                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:<br/>            exception_class = InvalidSelectorException<br/>        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:<br/>            exception_class = ElementNotSelectableException<br/>        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:<br/>            exception_class = ElementNotInteractableException<br/>        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:<br/>            exception_class = InvalidCookieDomainException<br/>        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:<br/>            exception_class = UnableToSetCookieException<br/>        elif status in ErrorCode.TIMEOUT:<br/>            exception_class = TimeoutException<br/>        elif status in ErrorCode.SCRIPT_TIMEOUT:<br/>            exception_class = TimeoutException<br/>        elif status in ErrorCode.UNKNOWN_ERROR:<br/>            exception_class = WebDriverException<br/>        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:<br/>            exception_class = UnexpectedAlertPresentException<br/>        elif status in ErrorCode.NO_ALERT_OPEN:<br/>            exception_class = NoAlertPresentException<br/>        elif status in ErrorCode.IME_NOT_AVAILABLE:<br/>            exception_class = ImeNotAvailableException<br/>        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:<br/>            exception_class = ImeActivationFailedException<br/>        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:<br/>            exception_class = MoveTargetOutOfBoundsException<br/>        elif status in ErrorCode.JAVASCRIPT_ERROR:<br/>            exception_class = JavascriptException<br/>        elif status in ErrorCode.SESSION_NOT_CREATED:<br/>            exception_class = SessionNotCreatedException<br/>        elif status in ErrorCode.INVALID_ARGUMENT:<br/>            exception_class = InvalidArgumentException<br/>        elif status in ErrorCode.NO_SUCH_COOKIE:<br/>            exception_class = NoSuchCookieException<br/>        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:<br/>            exception_class = ScreenshotException<br/>        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:<br/>            exception_class = ElementClickInterceptedException<br/>        elif status in ErrorCode.INSECURE_CERTIFICATE:<br/>            exception_class = InsecureCertificateException<br/>        elif status in ErrorCode.INVALID_COORDINATES:<br/>            exception_class = InvalidCoordinatesException<br/>        elif status in ErrorCode.INVALID_SESSION_ID:<br/>            exception_class = InvalidSessionIdException<br/>        elif status in ErrorCode.UNKNOWN_METHOD:<br/>            exception_class = UnknownMethodException<br/>        else:<br/>            exception_class = WebDriverException<br/>        if not value:<br/>            value = response[&#x27;value&#x27;]<br/>        if isinstance(value, str):<br/>            raise exception_class(value)<br/>        if message == &quot;&quot; and &#x27;message&#x27; in value:<br/>            message = value[&#x27;message&#x27;]<br/>    <br/>        screen = None  # type: ignore[assignment]<br/>        if &#x27;screen&#x27; in value:<br/>            screen = value[&#x27;screen&#x27;]<br/>    <br/>        stacktrace = None<br/>        st_value = value.get(&#x27;stackTrace&#x27;) or value.get(&#x27;stacktrace&#x27;)<br/>        if st_value:<br/>            if isinstance(st_value, str):<br/>                stacktrace = st_value.split(&#x27;\n&#x27;)<br/>            else:<br/>                stacktrace = []<br/>                try:<br/>                    for frame in st_value:<br/>                        line = self._value_or_default(frame, &#x27;lineNumber&#x27;, &#x27;&#x27;)<br/>                        file = self._value_or_default(frame, &#x27;fileName&#x27;, &#x27;&lt;anonymous&gt;&#x27;)<br/>                        if line:<br/>                            file = &quot;%s:%s&quot; % (file, line)<br/>                        meth = self._value_or_default(frame, &#x27;methodName&#x27;, &#x27;&lt;anonymous&gt;&#x27;)<br/>                        if &#x27;className&#x27; in frame:<br/>                            meth = &quot;%s.%s&quot; % (frame[&#x27;className&#x27;], meth)<br/>                        msg = &quot;    at %s (%s)&quot;<br/>                        msg = msg % (meth, file)<br/>                        stacktrace.append(msg)<br/>                except TypeError:<br/>                    pass<br/>        if exception_class == UnexpectedAlertPresentException:<br/>            alert_text = None<br/>            if &#x27;data&#x27; in value:<br/>                alert_text = value[&#x27;data&#x27;].get(&#x27;text&#x27;)<br/>            elif &#x27;alert&#x27; in value:<br/>                alert_text = value[&#x27;alert&#x27;].get(&#x27;text&#x27;)<br/>            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here<br/>&gt;       raise exception_class(message, screen, stacktrace)<br/><span class="error">E       selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element &lt;input data-role=&quot;none&quot; readonly=&quot;readonly&quot; type=&quot;text&quot; autocomplete=&quot;false&quot; name=&quot;flight_origin_date&quot; placeholder=&quot; &quot; id=&quot;BE_flight_origin_date&quot; class=&quot;custom-date-input BE_flight_origin_date&quot;&gt; is not clickable at point (250, 259). Other element would receive the click: &lt;li class=&quot;ac_over&quot;&gt;...&lt;/li&gt;</span><br/><span class="error">E         (Session info: chrome=98.0.4758.102)</span><br/><span class="error">E       Stacktrace:</span><br/><span class="error">E       Backtrace:</span><br/><span class="error">E       	Ordinal0 [0x004169A3+2582947]</span><br/><span class="error">E       	Ordinal0 [0x003AA6D1+2139857]</span><br/><span class="error">E       	Ordinal0 [0x002A3A98+1063576]</span><br/><span class="error">E       	Ordinal0 [0x002D636F+1270639]</span><br/><span class="error">E       	Ordinal0 [0x002D45B7+1263031]</span><br/><span class="error">E       	Ordinal0 [0x002D220B+1253899]</span><br/><span class="error">E       	Ordinal0 [0x002D0FF9+1249273]</span><br/><span class="error">E       	Ordinal0 [0x002C6B53+1207123]</span><br/><span class="error">E       	Ordinal0 [0x002E854C+1344844]</span><br/><span class="error">E       	Ordinal0 [0x002C6524+1205540]</span><br/><span class="error">E       	Ordinal0 [0x002E86A4+1345188]</span><br/><span class="error">E       	Ordinal0 [0x002F834A+1409866]</span><br/><span class="error">E       	Ordinal0 [0x002E8366+1344358]</span><br/><span class="error">E       	Ordinal0 [0x002C5176+1200502]</span><br/><span class="error">E       	Ordinal0 [0x002C6066+1204326]</span><br/><span class="error">E       	GetHandleVerifier [0x005BBE02+1675858]</span><br/><span class="error">E       	GetHandleVerifier [0x0067036C+2414524]</span><br/><span class="error">E       	GetHandleVerifier [0x004ABB01+560977]</span><br/><span class="error">E       	GetHandleVerifier [0x004AA8D3+556323]</span><br/><span class="error">E       	Ordinal0 [0x003B020E+2163214]</span><br/><span class="error">E       	Ordinal0 [0x003B5078+2183288]</span><br/><span class="error">E       	Ordinal0 [0x003B51C0+2183616]</span><br/><span class="error">E       	Ordinal0 [0x003BEE1C+2223644]</span><br/><span class="error">E       	BaseThreadInitThunk [0x775DFA29+25]</span><br/><span class="error">E       	RtlGetAppContainerNamedObjectPath [0x77C37A9E+286]</span><br/><span class="error">E       	RtlGetAppContainerNamedObjectPath [0x77C37A6E+238]</span><br/><br/>..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\errorhandler.py:243: ElementClickInterceptedException<br/> -----------------------------Captured stderr setup------------------------------ <br/>

====== WebDriver manager ======
Current google-chrome version is 98.0.4758
Get LATEST chromedriver version for 98.0.4758 google-chrome
Driver [C:\Users\Hp\.wdm\drivers\chromedriver\win32\98.0.4758.102\chromedriver.exe] found in cache
<br/> -------------------------------Captured log setup------------------------------- <br/>INFO     WDM:logger.py:26 

INFO     WDM:logger.py:26 ====== WebDriver manager ======
INFO     WDM:logger.py:26 Current google-chrome version is 98.0.4758
INFO     WDM:logger.py:26 Get LATEST chromedriver version for 98.0.4758 google-chrome
INFO     WDM:logger.py:26 Driver [C:\Users\Hp\.wdm\drivers\chromedriver\win32\98.0.4758.102\chromedriver.exe] found in cache<br/> ------------------------------Captured stdout call------------------------------ <br/>1
No match found for the search
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     LaunchPage:yatra_launch_page.py:55 Clicked on going to .
INFO     LaunchPage:yatra_launch_page.py:58 Typed text into going to field successfully.<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">testcases/test_search_flights.py::TestSearchAndVerifyFilter::test_search_flights_1_stop_2___New____MLB____27_02_2022____2_Stops__</td>
          <td class="col-duration">36.64</td>
          <td class="col-links"><a class="url" href="https://www.yatra.com/" target="_blank">URL</a> </td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div><div><img src="testcases/test_search_flights.py_TestSearchAndVerifyFilter_test_search_flights_1_stop_2___New____MLB____27_02_2022____2_Stops__.png" alt="screenshot" style="width:304px;height:228px;" onclick="window.open(this.src)" align="right"/></div></div>
            <div class="log">self = &lt;testcases.test_search_flights.TestSearchAndVerifyFilter testMethod=test_search_flights_1_stop_2___New____MLB____27_02_2022____2_Stops__&gt;<br/>going_from_origin = &#x27;New&#x27;, going_to_dest = &#x27;MLB&#x27;, going_depart_date = &#x27;27/02/2022&#x27;, stop_filter = &#x27;2 Stops&#x27;<br/><br/>    @data(*Utils.read_data_from_csv(&quot;C:\\Users\\Hp\\PycharmProjects\\TestFrameworkMentor\\testdata\\test_csv_data.csv&quot;))<br/>    @unpack<br/>    def test_search_flights_1_stop(self, going_from_origin, going_to_dest, going_depart_date, stop_filter):<br/>        search_flight = self.launch_page.searchFlight(going_from_origin, going_to_dest, going_depart_date)<br/>        # scrolling the page<br/>        self.launch_page.page_scroll()<br/>        search_flight.filter_flight_by_stop(stop_filter)<br/>        all_stops = search_flight.get_search_flight_results()<br/>        self.log.info(len(all_stops))<br/>&gt;       self.ut.UtilitiesList(all_stops, stop_filter)<br/><br/>testcases\test_search_flights.py:41: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>utilities\utils.py:19: in UtilitiesList<br/>    self.assert_all()<br/>..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py:139: in assert_all<br/>    self.fail(&#x27;&#x27;.join(failure_output))<br/><span class="error">E   AssertionError: ++++ soft assert failure details follow below ++++</span><br/><span class="error">E   </span><br/><span class="error">E   ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++</span><br/><span class="error">E   The following 3 failures were found in &quot;wrapper&quot; (utilities.utils.Utils):</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 1 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;2 Stops&#x27;</span><br/><span class="error">E   + 2 Stops</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [1/3] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 2 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;2 Stops&#x27;</span><br/><span class="error">E   + 2 Stops</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [2/3] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 3 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;2 Stops&#x27;</span><br/><span class="error">E   + 2 Stops</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [3/3] +-</span><br/> -----------------------------Captured stderr setup------------------------------ <br/>

====== WebDriver manager ======
Current google-chrome version is 98.0.4758
Get LATEST chromedriver version for 98.0.4758 google-chrome
Driver [C:\Users\Hp\.wdm\drivers\chromedriver\win32\98.0.4758.102\chromedriver.exe] found in cache
<br/> -------------------------------Captured log setup------------------------------- <br/>INFO     WDM:logger.py:26 

INFO     WDM:logger.py:26 ====== WebDriver manager ======
INFO     WDM:logger.py:26 Current google-chrome version is 98.0.4758
INFO     WDM:logger.py:26 Get LATEST chromedriver version for 98.0.4758 google-chrome
INFO     WDM:logger.py:26 Driver [C:\Users\Hp\.wdm\drivers\chromedriver\win32\98.0.4758.102\chromedriver.exe] found in cache<br/> ------------------------------Captured stdout call------------------------------ <br/>1
Melbourne (MLB)
Melbourne Regional
United States
391
The text is: 2 Stops
test pass
The text is: 2 Stops
test pass
The text is: 
test fail
The text is: 
test fail
The text is: 2 Stops
test pass
The text is: 
test fail
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     LaunchPage:yatra_launch_page.py:55 Clicked on going to .
INFO     LaunchPage:yatra_launch_page.py:58 Typed text into going to field successfully.
WARNING  SearchFlightResults:search_flights_result_page.py:50 Please provide the valid filter option
INFO     TestSearchAndVerifyFilter:test_search_flights.py:40 6<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">testcases/test_search_flights.py::TestSearchAndVerifyFilter::test_search_flights_1_stop_3___New____JFK____03_03_2022____1_Stop__</td>
          <td class="col-duration">54.88</td>
          <td class="col-links"><a class="url" href="https://www.yatra.com/" target="_blank">URL</a> </td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div><div><img src="testcases/test_search_flights.py_TestSearchAndVerifyFilter_test_search_flights_1_stop_3___New____JFK____03_03_2022____1_Stop__.png" alt="screenshot" style="width:304px;height:228px;" onclick="window.open(this.src)" align="right"/></div></div>
            <div class="log">self = &lt;testcases.test_search_flights.TestSearchAndVerifyFilter testMethod=test_search_flights_1_stop_3___New____JFK____03_03_2022____1_Stop__&gt;<br/>going_from_origin = &#x27;New&#x27;, going_to_dest = &#x27;JFK&#x27;, going_depart_date = &#x27;03/03/2022&#x27;, stop_filter = &#x27;1 Stop&#x27;<br/><br/>    @data(*Utils.read_data_from_csv(&quot;C:\\Users\\Hp\\PycharmProjects\\TestFrameworkMentor\\testdata\\test_csv_data.csv&quot;))<br/>    @unpack<br/>    def test_search_flights_1_stop(self, going_from_origin, going_to_dest, going_depart_date, stop_filter):<br/>        search_flight = self.launch_page.searchFlight(going_from_origin, going_to_dest, going_depart_date)<br/>        # scrolling the page<br/>        self.launch_page.page_scroll()<br/>        search_flight.filter_flight_by_stop(stop_filter)<br/>        all_stops = search_flight.get_search_flight_results()<br/>        self.log.info(len(all_stops))<br/>&gt;       self.ut.UtilitiesList(all_stops, stop_filter)<br/><br/>testcases\test_search_flights.py:41: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>utilities\utils.py:19: in UtilitiesList<br/>    self.assert_all()<br/>..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py:139: in assert_all<br/>    self.fail(&#x27;&#x27;.join(failure_output))<br/><span class="error">E   AssertionError: ++++ soft assert failure details follow below ++++</span><br/><span class="error">E   </span><br/><span class="error">E   ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++</span><br/><span class="error">E   The following 11 failures were found in &quot;wrapper&quot; (utilities.utils.Utils):</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 1 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [1/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 2 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [2/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 3 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [3/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 4 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [4/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 5 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [5/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 6 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [6/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 7 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [7/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 8 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [8/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 9 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [9/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 10 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [10/11] +-</span><br/><span class="error">E   </span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E    Failure 11 (&quot;wrapper&quot; method)</span><br/><span class="error">E   +----------------------+----------------------+----------------------+</span><br/><span class="error">E   Traceback (most recent call last):</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\utilities\utils.py&quot;, line 13, in UtilitiesList</span><br/><span class="error">E       self.soft_assert(self.assertEqual, stop.text, value)</span><br/><span class="error">E     File &quot;C:\Users\Hp\PycharmProjects\TestFrameworkMentor\testcases\test_search_flights.py&quot;, line 41, in test_search_flights_1_stop</span><br/><span class="error">E       self.ut.UtilitiesList(all_stops, stop_filter)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\ddt.py&quot;, line 191, in wrapper</span><br/><span class="error">E       return func(self, *args, **kwargs)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\lib\site-packages\softest\case.py&quot;, line 62, in soft_assert</span><br/><span class="error">E       assert_method(*arguments, **keywords)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 837, in assertEqual</span><br/><span class="error">E       assertion_func(first, second, msg=msg)</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 1217, in assertMultiLineEqual</span><br/><span class="error">E       self.fail(self._formatMessage(msg, standardMsg))</span><br/><span class="error">E     File &quot;C:\Users\Hp\AppData\Local\Programs\Python\Python310\Lib\unittest\case.py&quot;, line 667, in fail</span><br/><span class="error">E       raise self.failureException(msg)</span><br/><span class="error">E   AssertionError: &#x27;&#x27; != &#x27;1 Stop&#x27;</span><br/><span class="error">E   + 1 Stop</span><br/><span class="error">E   </span><br/><span class="error">E   -+ [11/11] +-</span><br/> -----------------------------Captured stderr setup------------------------------ <br/>

====== WebDriver manager ======
Current google-chrome version is 98.0.4758
Get LATEST chromedriver version for 98.0.4758 google-chrome
Driver [C:\Users\Hp\.wdm\drivers\chromedriver\win32\98.0.4758.102\chromedriver.exe] found in cache
<br/> -------------------------------Captured log setup------------------------------- <br/>INFO     WDM:logger.py:26 

INFO     WDM:logger.py:26 ====== WebDriver manager ======
INFO     WDM:logger.py:26 Current google-chrome version is 98.0.4758
INFO     WDM:logger.py:26 Get LATEST chromedriver version for 98.0.4758 google-chrome
INFO     WDM:logger.py:26 Driver [C:\Users\Hp\.wdm\drivers\chromedriver\win32\98.0.4758.102\chromedriver.exe] found in cache<br/> ------------------------------Captured stdout call------------------------------ <br/>1
New York (JFK)
John F Kennedy
United States
391
The text is: 1 Stop
test pass
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 
test fail
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 
test fail
The text is: 
test fail
The text is: 1 Stop
test pass
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 
test fail
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 1 Stop
test pass
The text is: 
test fail
The text is: 1 Stop
test pass
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     LaunchPage:yatra_launch_page.py:55 Clicked on going to .
INFO     LaunchPage:yatra_launch_page.py:58 Typed text into going to field successfully.
WARNING  SearchFlightResults:search_flights_result_page.py:39 selected flights with 1 stop.
INFO     TestSearchAndVerifyFilter:test_search_flights.py:40 21<br/></div></td></tr></tbody></table></body></html>